packages/sanity/src/core/studio/components/navbar/search/components/SearchPopover.tsx

```diff
@@ -20,9 +20,14 @@ import {RecentSearches} from './recentSearches/RecentSearches'
 import {SearchHeader} from './SearchHeader'
 import {SearchResults} from './searchResults/SearchResults'
 
+/**
+ * @internal
+ */
 export interface SearchPopoverProps {
   disableFocusLock?: boolean
+  disableIntentLink?: boolean
   onClose: () => void
+  onItemSelect?: (item: {documentId: string; documentType: string}) => void
```

Comment:
It's very small, but I wondered whether it'd be worth lifting this into a reusable type to avoid repeating it:

```ts
type ItemSelectHandler = (item: {documentId: string; documentType: string}) => void
```

Reply:
I also wonder whether it makes sense to type `item` as `Pick<SanityDocumentLike, '_id' | '_type'>`.

It would result in this:

```ts
{
  _id: string
  _type: string
}
```

This might help keep things consistent across the codebase, but I'm not sure whether `onItemSelect` is intended to handle items _other than_ documents in the future.

Reply:
Thanks, updated!